company_today/data/ir_cron.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+SPDX-FileCopyrightText: 2022 Coop IT Easy SCRLfs
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<odoo>
+    <record id="compute_today" model="ir.cron">
+        <field name="name">Set today's date on the company</field>
+        <field name="model_id" ref="model_res_company" />
+        <field name="code">model.cron_update_today</field>
+        <field name="active" eval="True" />
+        <field name="interval_number">1</field>
+        <field name="interval_type">minutes</field>
```

Comment:
does it run every minute, or am i missing something? 😊 once per day is enough, no?

Reply:
Then you'd have to make sure it runs at 00:01 am and I don't think you can do this with xml data.

Reply:
it will evaluate on any day to monday, because of `timedelta(days=-datetime.now().weekday(), weeks=1))` which will subtract the number of days between the current day and monday.

Reply:
do you mean that it will set the first run on a monday morning, and that after that it will run at the same time every week just because of the interval. that would work, but daylight saving time would mess things up for half of the year. 😒

Reply:
do you mean that it will set the first run on a monday morning, and that after that it will run at the same time every week just because of the interval. that would work, but daylight saving time would mess things up for half of the year. 😒

Reply:
do you mean that it will set the first run on a monday morning, and that after that it will run at the same time every week just because of the interval. that would work, but daylight saving time would mess things up for half of the year. 😒

Reply:
Correct. It would also mess up when the server is down at 00:00 and up again at, say, 3:00. It will notice that `nextcall` is in the past, run the cron, and then set the next interval 24 hours from 3:00.